docs/README.md

```diff
@@ -79,21 +79,50 @@ For links the path is relative to the base path (i.e. `/tools/vscode/`). The url
 
 ## How to generate Change Log
 
-### Install github-changelog-generator
+### Option 1: github-changelog-generator
+This tool is finnicky. If you can't get it working, try Option 2 instead.
+#### Install github-changelog-generator
 
 Instructions: https://github.com/github-changelog-generator/github-changelog-generator/blob/master/README.md
 
-### Commands
+#### Commands
 
 ```bash
 export CHANGELOG_GITHUB_TOKEN="<<Github Token>>"
 github_changelog_generator -u forcedotcom -p sfdx-scanner -f %m-%d-%Y --no-author --exclude-tags-regex "tag-test*" --no-verbose --since-tag vX.Y.Z
 
 ```
 
-### Note:
+#### Note:
 Copy the information for the release you are making to the top of the release-information.md
 
+### Option 2: pychangelog
+
+#### Prerequisites
+
+- Install [Python3](https://www.python.org/downloads/)
+- Install [Poetry](https://python-poetry.org/docs/#installation).
+  - If the `curl` script fails with an error about SSL certificate verification:
+    - If you're on OSX, you can resolve this by going to `~/Applications/Python 3.11` and running the `Install Certificates.command` located there.
+    - Haven't encountered this on Windows yet. If we do, a solution will go here.
+- Set up a [Github Access Token](https://github.com/settings/tokens).
+
+#### Using pychangelog
+
+Installation and general usage instructions are found [here](https://github.com/rero/pychangelog).
+
+You'll want to set the following values in the `config.ini` file:
+- `user = forcedotcom`
+- `repo = sfdx-scanner`
+- `merging_branch = dev`
+- `from_tag = [tag used for last published version]`
+- `to_tag = [tag to be used for next planned version]`
+
+Running `pychangelog` will create a list of all the PRs/Issues that were merged/closed since the last release.
+They'll be in a single list, but they're properly ordered so you can easily copy-paste them into the release notes' "Merged pull requests" and "Closed issues" sections.
+
+NOTE: PR's will have a `(by @author)` suffix. You'll want to delete that.
```

Comment:
pychangelog is accepting external PRs. It would be trivial to create a PR that adds a `--no-author` flag so we don't have to do that manually. If we can get that merged, this headache would go away.